.gitattributes

```diff
@@ -1 +1,4 @@
 * text=auto eol=lf
+# Denote all files that are truly binary and should not be modified.
+*.png binary
```

Comment:
I don't get it that you need to do this: https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/.gitattributes

Reply:
But with this the build passed. Without that it didn't. :confused:

Reply:
Btw the build result shouldn't be `continuous-integration/jenkins/pr-merge`? It is `continuous-integration/jenkins/branch` instead. Also the incrementals check is not returned by Jenkins. I think it is not archiving artifacts for some reason. In the deploy stage it says `bad http request` - https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fgitlab-branch-source-plugin/detail/hotfix%2Fgitlab-icons/9/pipeline/256

Reply:
@casz Based on the github example here - https://help.github.com/en/articles/configuring-git-to-handle-line-endings, it suggests to specifies binaries to skip, idk if it is necessary tho.